src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -2073,6 +2125,46 @@ public String getCharacterInformationScreen() {
 		
 		}
 		
+		if(this.isPlayer() && this.getOwner() != null && this.getOwner().getRules().size() > 0)
+		{
+			infoScreenSB.append("</br>"
+					+ "<h4>Master's Orders</h4>");
+			
+			boolean playerIsObeyingRules = true;
+			if(!this.getOwner().isPlayerObeyingRules())
+			{
+				playerIsObeyingRules = false;
+				infoScreenSB.append("<p>"
+										+ "You are <b style='color:" + Colour.GENERIC_MINOR_BAD.toWebHexString()+";'>disobeying</b> some of the rules"
+										+ " and as a result, your positive obedience gains are halted."
+									+ "</p>");
+			}
+			
+			infoScreenSB.append("<p>");
+			
+			for(PlayerSlaveryRule rule : this.getOwner().getRules())
+			{
+				String rulePerformanceValueString;
```

Comment:
Use the if statement to pick the color, then only build the whole string in a single place. Try not to repeat yourself if possible and easy.

Reply:
Fair enough, swapped out.